readme.md

```diff
@@ -150,6 +151,7 @@ Click the type names for complete docs.
 - [`UnionToIntersection`](source/union-to-intersection.d.ts) - Convert a union type to an intersection type.
 - [`LiteralToPrimitive`](source/literal-to-primitive.d.ts) - Convert a [literal type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types) to the [primitive type](source/primitive.d.ts) it belongs to.
 - [`Stringified`](source/stringified.d.ts) - Create a type with the keys of the given type changed to `string` type.
+- [`IsEmptyObject`](source/empty-object.d.ts) - Returns a `boolean` whether the type is strictly equal to `{}`.
```

Comment:
Just saying `{}` is ambiguous because it's both a JS literal and TS type.

https://stackoverflow.com/a/52193484/64949